js/okex.js

```diff
@@ -790,69 +790,70 @@ module.exports = class okex extends Exchange {
                 symbol = symbol + '-' + ymd;
             }
             if (option) {
-                const strikePrice = this.safeString (market, 'stk');
-                const optionType = this.safeString (market, 'optType');
+                strikePrice = this.safeString (market, 'stk');
+                optionType = this.safeString (market, 'optType');
                 symbol = symbol + '-' + strikePrice + '-' + optionType;
+                optionType = (optionType === 'P') ? 'put' : 'call';
             }
         }
         const tickSize = this.safeString (market, 'tickSz');
-        const precision = {
-            'amount': this.safeNumber (market, 'lotSz'),
-            'price': this.parseNumber (tickSize),
-        };
         const minAmountString = this.safeString (market, 'minSz');
         const minAmount = this.parseNumber (minAmountString);
         let minCost = undefined;
         if ((minAmount !== undefined) && (tickSize !== undefined)) {
```

Comment:
We don't need to check `minAmount` and `tickSize` for `undefined` here, cause `stringMul` and `parseNumber` can handle `undefined` values.